### DISCUSSION TITLE
Always ask for Watchlist.xlsx although it is in right directory

### DISCUSSION THREAD

**Discussion by rami202020** [pranjal-joshi/Screeni-py#238] [General]
I checked and found that the file exits although it exists, and error code is exactly as follows:
![image](https://github.com/pranjal-joshi/Screeni-py/assets/76410008/4a649db6-7a9b-4188-bf87-32600dcb07a8)
![error](https://github.com/pranjal-joshi/Screeni-py/assets/76410008/a933fb01-a20d-4604-9291-329ff3e79cfb)

**Comment by pranjal-joshi** [comment]
File name must be exactly `watchlist.xlsx` and not `watchlist_template.xlsx`
